.circleci/config.yml

```diff
@@ -163,6 +163,10 @@ workflows:
       - publish:
           filters:
             tags:
-              only: /^v.*/
+              only: /v[\.0-9]+.*/
+            branches:
+              only:
+                - master
+                - /release\/.*/
```

Comment:
A little escape hatch and also to have better releasing from branches if needed.